fdbclient/NativeAPI.actor.cpp

```diff
@@ -296,9 +295,6 @@ ACTOR static Future<Void> transactionInfoCommitActor(Transaction *tr, std::vecto
 			return Void();
 		}
 		catch (Error& e) {
-			retryCount++;
-			if (retryCount == 10)
```

Comment:
where do we handle the retry logic now? At some transaction class now?

Reply:
Oh good catch, this was from me trying to delete something else. I'll add this logic back.